proposals/nnnn-structured-concurrency.md

```diff
@@ -167,7 +167,7 @@ This proposal doesn't propose solutions for all of these, but early investigatio
 
 ### Tasks
 
-A task is the basic unit of concurrency in the system. It is the analogue of a thread for asynchronous functions. That is:
+A task is the basic unit of concurrency in the system. It is the analogue of a thread for synchronous functions. That is:
```

Comment:
Hey @dduan, actually watching your review of the proposals now (thanks, was quite fun/interesting to learn where more explanations are needed!) 🙂  

This could be more of an english question I guess, but in my mind the wording is correct however it definitely can be confusing so let's improve it.

Today we're saying:

> Task [...] It is the analogue of a thread for asynchronous functions. That is:

By which we mean:

- since thread is for synchronous function land,
- the analogue for thread in the asynchronous world, is task.

I see where the confusion though can come from. How about being extremely explicit about it:

```suggestion
A task is the basic unit of concurrency in the system. Every asynchronous function is executing in a task. In other words, a _task_ is to _asynchronous functions_, what a _thread_ is to _synchronous functions_. That is:
```

cc @rjmccall @DougGregor  wdyt, is this more clear/explicit?

Reply:
Whoa thanks for watching 😳. I actually closed this PR and reopened it because I saw the other interpretation as well. I think the new wording is great!